python/az/aro/azext_aro/tests/latest/test_aro_helpers.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+from azure.cli.command_modules.aro.custom import generate_random_id
+
+
+class TestGenerateRandomIdHelper(unittest.TestCase):
+    def test_random_id_length(self):
+        random_id = generate_random_id()
+        self.assertEqual(len(random_id), 8)
+
+    def test_random_id_starts_with_letter(self):
+        random_id = generate_random_id()
+        self.assertTrue(random_id[0].isalpha())
```

Comment:
It does look like you implement this , but just saying "it is random" so the test could hide this ;-)
If you are very enthusiastic you *could* mock.patch random.choice and have a test that returns every option passed to it